backend/domain/account/account_crud.py

```diff
@@ -0,0 +1,81 @@
+from sqlalchemy.orm import Session
+from domain.account.account_schema import (
+    AccountCreate,
+    AccountUpdate,
+)
+from models import User, Account
+import uuid
+from datetime import datetime, timedelta, timezone
+from starlette import status
+from fastapi import HTTPException
+
+
+def create_account(
+    db: Session,
+    account_create: AccountCreate,
+    user: User,
+) -> Account:
+    """
+    Creates new account
+    """
+    db_account = Account(
+        id=str(uuid.uuid4()),
+        account_name=account_create.institution,
+        account_type=account_create.account_type,
+        current_balance=account_create.current_balance,
+        user_id=user.id,
+        user=user,
+    )
+
+    db.add(db_account)
+    db.commit()
+    return db.get_account_by_name(db, account_create.institution)
```

Comment:
db object does not have get_account_by_name() method. It is a stand alone function. This need to be fixed throughout the module.